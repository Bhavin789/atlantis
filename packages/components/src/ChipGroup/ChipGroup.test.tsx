import React from "react";
import { cleanup, render } from "@testing-library/react";
import { ChipGroup } from ".";

afterEach(cleanup);

it("renders an empty Chip Group", () => {
  const { container } = render(<ChipGroup chips={[]} />);
  expect(container).toMatchSnapshot();
});
