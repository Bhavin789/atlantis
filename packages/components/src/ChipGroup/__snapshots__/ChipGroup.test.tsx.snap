// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an empty Chip Group 1`] = `
<div>
  <span
    style="display: flex; gap: var(--space-base);"
  />
</div>
`;
